explorer/app/components/Index/common/constants.ts

```diff
@@ -20,13 +20,22 @@ const {
   DIAGNOSIS,
   DISEASE_DONOR,
   DISEASE_INDICATION,
+  DISEASE_STATUS_DONOR,
+  DISEASE_STATUS_SPECIMEN,
+  FILE_FORMAT,
   FOCUS_DISEASE,
   LIBRARY_CONSTRUCTION_APPROACH,
   LIBRARY_PREPARATION,
+  MODEL_ORGAN,
+  NUCLEIC_ACID_SOURCE,
+  ORGAN_PART,
   ORGANISM_TYPE,
+  PAIRED_END,
```

Comment:
Given Paired End is a set of three possible values, revert here and elsewhere all Paired End related changes. Will will always show the values, stringyified...